### ISSUE TITLE
a wish: add subsetting mechanism - one set includes the other

### ISSUE THREAD

**Issue by ardabro** [proginosko/LeechBlockNG#461]
It would be very convenient.

**Comment by proginosko** [comment]
Please explain more about how this would work. Do you mean just the list of sites?

**Comment by ardabro** [comment]
Yeah. It is all about the list of sites. There are multiple rule sets. They have independent site lists.
I'd like to specify, that setA, let's call it MAX inherits all site addresses from setB, let's call it MODERATE, and of course adds more addresses on its own list. Then, when I find a new very toxic site I want to block all the time, I can add it once to the list of the setB and ensure, that it will also affect all (usually more rigorous) sets inheriting from it.
It could significantly reduce site address duplications and headaches with maintenance.
